### PR TITLE
Write message when ctrl-c is received

### PR DESCRIPTION
**Pull request by thebriando**
# ↪️ Pull Request
Writes a message to the console when `ctrl-c` is received to notify the user that Parcel is shutting down.

## 🚨 Test instructions

Run Parcel in a directory and exit with `ctrl-c` to see the message that is outputted.

## ✔️ PR Todo

- [ ] Added/updated unit tests for this change
- [ ] Filled out test instructions (In case there aren't any unit tests)
- [ ] Included links to related issues/PRs

### CODE REVIEW

**Review by padmaia — APPROVED** [review]
Looks good to me, the only thing I can think of is maybe adding a chalk color? @thebriando could you maybe provide a picture to show what it looks like?

**Comment by thebriando** [comment]
@padmaia Does this look ok? 
image

File: packages/core/parcel/src/cli.js [hunk 1]
```diff
@@ -223,6 +223,9 @@ async function run(entries: Array<string>, command: any) {
           // We don't use the SIGINT event for this because when run inside yarn, the parent
           // yarn process ends before Parcel and it appears that Parcel has ended while it may still
           // be cleaning up. Handling events from stdin prevents this impression.
+          INTERNAL_ORIGINAL_CONSOLE.log(
+            chalk.bold.yellowBright('Parcel is shutting down'),
```
Comment (wbinnssmith) [inline packages/core/parcel/src/cli.js:227]: nit: Maybe "Parcel is shutting down..."?

**Comment by mischnic** [comment]
Should this message be displayed with a small delay (via setTimeout)? I haven't run into a situation yet where Parcel seemed to hang after pressing Ctlr-C (as in: my projects weren't big enough 😄 ).

**Comment by thebriando** [comment]
@mischnic Hmm how long should the timeout be?

**Comment by mischnic** [comment]
I would say long enough so that you don't ask yourself if Parcel crashed for big projects and short enough that you don't even see the message for small(er) projects. Probably below 1 second?

**Pull request merged**